skimage/measure/_marching_cubes_lewiner_cy.pyx

```diff
@@ -999,6 +997,79 @@ def marching_cubes(float [:, :, :] im not None, double isovalue, LutProvider lut
     return cell.get_vertices(), cell.get_faces(), cell.get_normals(), cell.get_values()
 
 
+def marching_cubes_masked(float [:, :, :] im not None, double isovalue, LutProvider luts, int st=1, int classic=0,
```

Comment:
Is there any way to avoid this code duplication?  I.e., have a check inside of the existing marching cubes function instead?

Reply:
yes, I was able to do it. Sorry I am not too familiar with cython so I just went to the easy route. Now is much better